libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -66,52 +66,52 @@ const AP_Param::GroupInfo AP_MotorsHeli::var_info[] = {
     AP_GROUPINFO("SV_MAN",  6, AP_MotorsHeli, _servo_mode, SERVO_CONTROL_MODE_AUTOMATED),
 
     // @Param: RSC_SETPOINT
-    // @DisplayName: External Motor Governor Setpoint
-    // @Description: PWM in microseconds passed to the external motor governor when external governor is enabled
-    // @Range: 0 1000
+    // @DisplayName: Electric ESC Throttle Setting
+    // @Description: Throttle signal percent for electric helicopters when a governor is used in the ESC
+    // @Range: 0 100
     // @Units: PWM
-    // @Increment: 10
+    // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("RSC_SETPOINT", 7, AP_MotorsHeli, _rsc_setpoint, AP_MOTORS_HELI_RSC_SETPOINT),
 
     // @Param: RSC_MODE
-    // @DisplayName: Rotor Speed Control Mode
-    // @Description: Determines the method of rotor speed control
-    // @Values: 1:Ch8 Input, 2:SetPoint, 3:Throttle Curve, 4:Governor
+    // @DisplayName: Throttle Control Mode
```

Comment:
I don't think this should be named "Throttle Control Mode".  This is the method by which we control the rotor speed.  We use the throttle to accomplish that but this is the method by which we control rotor speed.

Reply:
I disagree because rotor speed is actually independent of the throttle due to the autorotation clutch, and only directly connected under power. If you look at the values that can be selected, this is actually selecting the throttle control mode for the prime mover. This was based on the recent discussion on the forum with a user trying to figure out how to control the throttle with the collective lever. The current descriptions for setup of the system suck. This whole system was designed by computer programmers and engineers that are totally out of touch with the end users. I am aiming to change that by using the metadata to provide descriptions and Display Names that users understand.

Reply:
I still think it should be rotor speed control mode.  Whether it’s open loop or closed loop, you are using the throttle to control the rotor speed. Add a better description and improve the wiki so users understand this sets the throttle schedule or rotor governor.

Reply:
OK, I'll change this, but I'll wait for your commit on the moving of the params to RSC first.